src/timeseriesinsights/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.0
+++++++
+* Breaking Change: use track2 SDK
```

Comment:
This doesn't seem like user-facing change. Maybe we can put actual changes here?